UKHomeOffice/cop-private-workflow-tasklist
# COP-1504 Update throttle to run without erroring

**Issue**
Error appearing in console when 'load next' was clicked on Cases page
```react-dom.production.min.js?ca5d:17 Uncaught TypeError: Expected a function
    at Function.throttle (lodash.js?2ef0:10928)
    at Object.loadNext (CasesPage.jsx?e339:161)
    at onClick (CaseResultsPanel.jsx?62df:156) 
```

**To test resolution**

- go to Cases
- search for 'Enhance' or something you know will produce more than 20 results
- click on Load Next button
- you should not see the above error in the console
